witchcraft-wasm-logger/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "witchcraft-wasm-logger"
+version = "0.1.0"
```

Comment:
We'll want to start this at 1.0.0 so autorelease's behavior lines up with cargo's semver assumptions.